sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTest.java

```diff
@@ -1644,21 +1647,6 @@ public void analyzeTasksPagination(HttpClient httpClient, TextAnalyticsServiceVe
         }, 22);
     }
 
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
```

Comment:
It will be no longer applicable since we don't allow user to set the `top` and `skip` values